guides/common/modules/proc_changing_the_content_source_of_a_host.adoc

```diff
@@ -0,0 +1,16 @@
+[id="Changing_the_Content_Source_of_a_Host{context}"]
+= Changing the Content Source of a Host
+
+A Content Source is a {SmartProxy} that a host consumes content from. Use this procedure to change the Content Source of a Host
+
+.Procedure
+. In the {ProjectWebUI}, navigate to *Hosts* > *All hosts*.
+. Select the check box of the host or hosts you want to change. 
+. From the *Select Action* list, select *Change Content Source*.
+. From the *Environment* list, select the new environment for your host.
+. From the *Content View* list, select the new Content View for your host.
+. From the *Content Source* list, select the new {SmartProxy} for your host.
+. Click *Change content source*. 
+
+You can either complete the content source change using remote execution or manually. To update configuration on host using remote execution, click on *Run job invocation*. To update the content source manually, enter the autogenerated commands in *Change content source* on the host.
```

Comment:
Please provide an xref to https://docs.theforeman.org/3.1/Managing_Hosts/index-foreman-el.html#executing-a-remote-job_managing-hosts